package.json

```diff
@@ -56,9 +56,12 @@
         "prepare": "husky install && git config feature.manyFiles true && node ./crates/swc_ecma_preset_env/scripts/copy-data.js",
         "artifacts": "napi artifacts --dist scripts/npm",
         "prepublishOnly": "tsc -d && napi prepublish -p scripts/npm --tagstyle npm",
-        "build": "tsc -d && napi build --platform --cargo-name node --js ./node-swc/src/binding.js --dts ./node-swc/src/binding.d.ts -p node --release",
-        "build:dev": "tsc -d && napi build --platform --cargo-name node --js ./node-swc/src/binding.js --dts ./node-swc/src/binding.d.ts -p node",
+        "pack": "wasm-pack",
         "build:ts": "tsc -d",
+        "build:wasm": "npm-run-all \"pack -- build ./crates/wasm --scope swc {1} -t {2} --features plugin\" --",
```

Comment:
Leaves existing `script/build.sh` as-is (for now), while makes it easy to local dev run wasm build.

`npm run build:wasm -- --debug|--release nodejs|web` places argument into placeholder.